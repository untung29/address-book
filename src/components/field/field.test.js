import React from "react";
import renderer from "react-test-renderer";
import Field from "./field.component";

it("<Field /> Snapshot Testing", () => {
  const FieldTest = renderer
    .create(<Field label="First Name" id="firstname" type="text" placeholder="First Name" />)
    .toJSON();
  expect(FieldTest).toMatchSnapshot();
});
