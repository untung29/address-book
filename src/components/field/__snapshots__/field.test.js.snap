// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Field /> Snapshot Testing 1`] = `
<div
  className="form-group"
>
  <input
    className="form-control"
    id="firstname"
    placeholder="First Name"
    required={true}
    type="text"
  />
</div>
`;
